dashboard/app/controllers/api/v1/pd/workshop_survey_foorm_submissions_controller.rb

```diff
@@ -52,13 +53,16 @@ def save_facilitator_questions(facilitator_answers, params)
     survey_submission_ids = []
     if facilitator_answers
       facilitator_answers.each do |_, data|
-        next unless data["facilitator_id"]
+        # data needs to have facilitator id
+        next unless data[FACILITATOR_ID]
+        # If data only contains metadata, do not store, as there were no answers for this facilitator.
+        next if data.except(FACILITATOR_ID, FACILITATOR_NAME, FACILITATOR_POSITION).nil_or_empty?
```

Comment:
Not necessary for this PR, but in the future maybe some way we can identify a group of metadata fields somewhere? I could imagine this list of metadata changing, then we start writing results when there isn't any real data.